src/main/java/com/revlearn/team1/dto/discussionBoard/DiscussionBoardDTO.java

```diff
@@ -0,0 +1,8 @@
+package com.revlearn.team1.dto.discussionBoard;
+
+public record DiscussionBoardDTO(
```

Comment:
HAHA last change i promise, can you rename this to be a Request or Response obj. I'm DEF going to forget which way its flowing


DiscussionBoardRequestDTO or DiscussionBoardResponseDTO or something like that

Reply:
I'm using Light, which has the discussion board id, course id (as a Long not Course), and title.
You had a good point about needing to change a lot of things if Course is updated

Reply:
Renamed it to DiscussionBoardDTO.  Request and Response will use this.